core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -186,6 +186,10 @@ func StartNewChainWithContracts(t *testing.T, nClients int) (*bind.TransactOpts,
 	linkEthFeedAddr, _, _, err := mock_v3_aggregator_contract.DeployMockV3AggregatorContract(owner, b, 18, big.NewInt(5_000_000_000_000_000))
 	require.NoError(t, err)
 
```

Comment:
I noticed we don't have additional tests that cover both the previous contract version & new contracts. My guess is this is gonna be flagged for lack of coverage by SonarQube. I think we need to write tests which cover both in order to get this PR merged.

Reply:
Actually, I think the SonarQube coverage requirement may have been temporarily paused for other reasons? Either way, I'd really prefer we test with both versions of the contract here just to be safe.
I'm also curious if we can simulate a coordinator contract upgrade in our tests.

Reply:
There is a test with an upgrade, though it doesn't go from version to version, because previously we only held on to one wrapper.

I think it's a good idea to change this test to actually move through breaking changes

Reply:
I took an attempt with this, but the types got messy again.
I'm opening a new ticket to ensure that this happens, even though it won't be in this PR.

Reply:
okay. In the mean time, we should properly test this process on staging.